atomkraft/test/trace.py

```diff
@@ -32,11 +32,8 @@ def test_trace(trace: PathLike, reactor: PathLike, keypath: str):
             "could not find Atomkraft project: are you in the right directory?"
         )
 
-    tests = os.path.join(root, "tests")
-    if not os.path.isdir(tests):
-        raise RuntimeError(
-            "Atomkraft tests directory not found: have you executed `atomkraft init`?"
-        )
+    tests = root / "tests"
+    tests.mkdir(exist_ok=True)
```

Comment:
I propose to make `tests` directory optional for a project. Create it on-the-fly when needed - as we use the traces directly via `atomkraft test traces`.

Reply:
As a short-term solution this is fine. As a long-term, we should automate the directory management similar to the [config file management](https://github.com/informalsystems/atomkraft/tree/dev/atomkraft/config).